applications/RomApplication/python_scripts/rom_solver.py

```diff
@@ -19,19 +20,28 @@ def __init__(self, model, custom_settings):
         @classmethod
         def GetDefaultParameters(cls):
             default_settings = KratosMultiphysics.Parameters("""{
+                "solving_strategy" : "Galerkin",
                 "rom_settings": {
                     "nodal_unknowns": [],
-                    "number_of_rom_dofs": 0
+                    "number_of_rom_dofs" : 10
```

Comment:
Why changing the defaults? Indeed, I think that we should keep the zero to throw an error if it is not specified by the user considering that this is something completely problem dependent.